config/schemas/config.yaml

```diff
@@ -0,0 +1,2854 @@
+        items:
+          $ref: '#/$defs/clusterid'
+        type: array
+      enabled:
+        default: false
+        type: boolean
+      monitoring:
+        properties:
+          enabled:
+            default: false
+            type: boolean
+        type: object
+    type: object
+  dex:
+    properties:
+      additionalKubeloginRedirects:
+        type: array
+      affinity:
+        $ref: '#/$defs/affinity'
+      enableStaticLogin:
+        default: true
+        type: boolean
+      expiry:
+        properties:
+          deviceRequests:
+            default: 5m
+            type: string
+          idToken:
+            default: 24h
+            type: string
+          refreshTokens:
+            properties:
+              absoluteLifetime:
+                default: 3960h
+                type: string
+              reuseInterval:
+                default: 3s
+                type: string
+              validIfNotUsedFor:
+                default: 2160h
+                type: string
+            type: object
+          signingKeys:
+            default: 6h
+            type: string
+        type: object
+      google:
+        properties:
+          groupSupport:
+            default: false
+            type: boolean
+        type: object
+      nodeSelector:
+        $ref: '#/$defs/nodeSelector'
+      replicaCount:
+        default: 2
+        type: number
+      resources:
+        $ref: '#/$defs/resources'
+      serviceMonitor:
+        properties:
+          enabled:
+            default: true
+            type: boolean
+        type: object
+      subdomain:
+        default: dex
+        type: string
+      tolerations:
+        $ref: '#/$defs/tolerations'
+      topologySpreadConstraints:
+        items:
+          properties:
+            labelSelector:
+              properties:
+                matchLabels:
+                  properties:
+                    app.kubernetes.io/name:
+                      default: dex
+                      type: string
+                  type: object
+              type: object
+            maxSkew:
+              default: 1
+              type: number
+            topologyKey:
+              default: kubernetes.io/hostname
+              type: string
+            whenUnsatisfiable:
+              default: DoNotSchedule
+              type: string
+          type: object
+        type: array
+    type: object
+  externalTrafficPolicy:
+    properties:
+      local:
+        default: false
+        type: boolean
+      whitelistRange:
+        properties:
+          global:
+            default: 0.0.0.0/0
+            type: string
+          dex:
+            type: boolean
+          harbor:
+            type: boolean
+          opensearch:
+            type: boolean
+          opensearchDashboards:
+            type: boolean
+          opsGrafana:
+            type: boolean
+          thanosReceiver:
+            type: boolean
+          userGrafana:
+            type: boolean
```

Comment:
All of these are supposed to be strings, and the default should be an empty one. :sweat_smile:

Reply:
Empty string being equivalent to `0.0.0.0/0` then?

Reply:
Effectively yes, although it won't render the actual annotation when it is completely empty.